### ISSUE TITLE
Consider adding minio component

### ISSUE THREAD

**Issue by invidian** [kinvolk/lokomotive#919]
To make testing Velero backups (#829) easier, we can consider adding a component for Minio (https://github.com/minio/charts/tree/master/minio), so we can configure Velero to store the backups there via restic.

**Comment by invidian** [comment]
>The place where you backup things, shouldn't it be outside your current cluster. I am assuming that velero and minio run in same cluster here?

Yes, it should be outside of the cluster, but for testing, it should be OK. Also, you can have cluster A with minio deployed, where backups from cluster B are send to.

**Comment by surajssd** [comment]
For testing yes, sure we can have it.
